cloudbuild.yaml

```diff
@@ -0,0 +1,23 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
+
+# this must be specified in seconds. If omitted, defaults to 600s (10 mins)
+timeout: 1200s
+# this prevents errors if you don't use both _GIT_TAG and _PULL_BASE_REF,
+# or any new substitutions added in the future.
+options:
+  substitution_option: ALLOW_LOOSE
+steps:
+  - name: gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20220609-2e4c91eb7e
+    entrypoint: make
+    env:
+    - REGISTRY=gcr.io/k8s-staging-cloud-provider-os
+    - VERSION=$_GIT_TAG
+    args:
+    - upload-images
+substitutions:
+  # _GIT_TAG will be filled with a git-based tag for the image, of the form
+  # vYYYYMMDD-hash, and can be used as a substitution
+  _GIT_TAG: '12345'
+  # _PULL_BASE_REF will contain the ref that was pushed to to trigger this
+  # build - a branch like 'master' or 'release-0.2', or a tag like 'v0.2'.
+  _PULL_BASE_REF: 'master'
```

Comment:
We're not using this. However, as it seems to be a standard substitution probably simpler to leave it in to reduce friction if we want to use it in the future.

Reply:
seems correct? CPO didn't switch to main currently, still using master as primary dev branch..